command/agent.go

```diff
@@ -30,6 +31,8 @@ import (
 	"github.com/hashicorp/vault/command/agent/auth/kerberos"
 	"github.com/hashicorp/vault/command/agent/auth/kubernetes"
 	"github.com/hashicorp/vault/command/agent/cache"
+	"github.com/hashicorp/vault/command/agent/cache/cachememdb"
+	"github.com/hashicorp/vault/command/agent/cache/cachepersist"
```

Comment:
Wonder if this should be `cacheboltdb` instead?

Reply:
Sure, I like that name.